parser/SqlLexer.lpp

```diff
@@ -120,6 +120,7 @@ unsigned_numeric_literal {exact_numeric_literal}|{approximate_numeric_literal}
 "between"          return TOKEN_BETWEEN;
 "bigint"           return TOKEN_BIGINT;
 "bit"              return TOKEN_BIT;
+"bloom_filter"     return TOKEN_BLOOM_FILTER_INDEX;
```

Comment:
We don't have a bloom filter index in master yet (although if one is in the works, that's awesome). We do, however, have a CSBTree index that we want to be able to use, so it would be good if we had a keyword for that so that we can specify such an index.